server/api/routes.py

```
@@ -10,11 +10,11 @@
 from datetime import datetime
 from time import sleep
 
-from core.routing.graph_builder import build_simplified_graph, load_graph_from_file, extract_subgraph
+from core.routing.graph_builder import  load_graph_from_file, extract_subgraph, visualize_dijkstra_points, visualize_astar_points
```

Flake8 (7.2.0)

[error] multiple spaces after keyword

(E271)